aemedge/styles/styles.css

```diff
@@ -554,6 +554,111 @@ main .section {
   }
 }
 
+.article-collection-wrapper, .content-list-wrapper {
+  margin-bottom: var(--udexSpacer56);
```

Comment:
IIRC by default all sections have bottom margin of 56px. Not sure where the rule is but why do we need to do it specifically for these containers?

Reply:
I added the spacing so that there would be vertical spacing between siblings of the same section. I can add some more specificity here to avoid doubling the padding if it is the last element of the section

Reply:
Maybe we should create a ticket to take a look at global section/block spacing to ensure a consistent approach. Happy to pick it up.